core/src/test/java/tachyon/client/TachyonFSTest.java

```diff
@@ -176,7 +176,6 @@ public void deleteFileTest() throws IOException {
     List<ClientWorkerInfo> workers = sTfs.getWorkersInfo();
     Assert.assertEquals(1, workers.size());
     Assert.assertEquals(WORKER_CAPACITY_BYTES, workers.get(0).getCapacityBytes());
-    Assert.assertEquals(0, workers.get(0).getUsedBytes());
```

Comment:
This is not the case, other asserts for workers.get(0).getUsedBytes() in this test later on will also fail.